### ISSUE TITLE
Problem with symbolic links

### ISSUE THREAD

**Issue by julien-f** [tivac/modular-css#105]
When a CSS file is accessed though a symlink (directly or indirectly), I get the following error:

```
Error: Node does not exists: <real path of the file>
    at Object.DepGraph.dependenciesOf (<project path>/node_modules/dependency-graph/lib/dep_graph.js:139:13)
    at Object.Processor.dependencies (<project path>/node_modules/modular-css/src/processor.js:151:35)
```

My use case: a module using a CSS module is in a `common` aliased as `node_modules` with a symlink to make it includable without specifying a relative path.

**Comment by tivac** [comment]
I'll take a look.

**Comment by tivac** [comment]
I just wrote up a test in the [`issues/105`](https://github.com/tivac/modular-css/tree/issues/105) branch, but it passes for me on windows. Would you be willing to try it locally?

1. Sync the branch
2. `npm i`
3. `ln -s test/specimens/issues/105/symlink.css test/specimens/issues/105/folder2.css`
4. `npm run mocha`

Does that work for you? Git on windows apparently can't submit symlinks, so I've got no simple way to add the symlink to the repo and test on Travis :angry:

**Comment by julien-f** [comment]
With or without the link I get:

```
  1) /issues /105 should be able to compose using a symlink:
     Error: timeout of 5000ms exceeded. Ensure the done() callback is being called in this test.
```

If I change the catch clause of the promise (line 31) by `done` I get:

```
  1) /issues /105 should be able to compose using a symlink:
     Unable to locate "./symlink.css" from "/tmp/tmp.KSqKdy0DJA/tmp/modular-css/test/specimens/issues/105"
```

After doing the correct link (`ln -s folder/2.css symlink.css` ^^) the test pass!

You can find the issue I was describing [there](https://github.com/julien-f/modular-css-issue-105) when doing a `npm run build`.

**Comment by tivac** [comment]
Checking out your repro repo, thanks!

**Comment by tivac** [comment]
@julien-f would you pull master and verify that #107 fixed your issue? If I don't hear from you in a day or two I'll publish to NPM anyways, but would be nice to get verification. Thanks again for the repo, that really helped narrow it down to the browserify plugin!

**Comment by julien-f** [comment]
It seems to be fixed, thank you!

**Comment by tivac** [comment]
Just published `modular-css@0.16.0` with this fix.
